docs/source/execute.rst

```diff
@@ -74,7 +74,7 @@ calling ``reproman run --list=orchestrators``.
 
 The main orchestrator choices are ``datalad-pair``,
 ``datalad-pair-run``, and ``datalad-local-run``. If the remote has
-DataLad available, you should go with one of the ``pair`` orchestrators.
+DataLad available, you should go with one of the ``datalad-pair*`` orchestrators.
```

Comment:
OK with me, but given the context of the previous sentence, I think my version was clear and easier on the eyes.

Reply:
I went for more specific to make it more explicit and future proof (if another non `datalad-` `pair`) appears, and also consistent with use of `*` glob around